clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java

```diff
@@ -66,7 +66,7 @@ public ClusterConnectionStates(long reconnectBackoffMs, long reconnectBackoffMax
     }
 
     /**
-     * Return true iff we can currently initiate a new connection. This will be the case if we are not
+     * Return true if we can currently initiate a new connection. This will be the case if we are not
```

Comment:
It means "if and only if".

Reply:
I need to improve my English, thanks for your explanation, I will close this PR.